requirements.txt

```diff
@@ -0,0 +1,4 @@
+aiohttp==1.0.5
+pytest==3.0.4
+requests==2.12.1
+responses==0.5.1
```

Comment:
I'd reconsider the version pinning. I see a world of pain, filled with issues and forks.

Assume for a sec Appo, our app developer, takes care of his dependencies, and pins them. Then he tries to install your library, and *boom* he can't, because for whatever reason he uses requests `2.12.1`, but 2.13. Or maybe he follows the [two-files approach from Kenneth](https://www.kennethreitz.org/essays/a-better-pip-workflow) or the `https://github.com/pypa/pipfile` to pin indirect dependencies, too, and a library that they're already using needs `2.12.2`. Can't use your lib.

Remember, you're building a library, not an app. Any assumption you're making, you're forcing onto your users.